plaid/link_token.go

```diff
@@ -0,0 +1,68 @@
+package plaid
+
+import (
+	"encoding/json"
+	"time"
+)
+
+type LinkTokenUser struct {
+	ClientUserID             string    `json:"client_user_id"`
+	LegalName                string    `json:"legal_name,omitempty"`
+	PhoneNumber              string    `json:"phone_number,omitempty"`
```

Comment:
do we want to optional some of these

Reply:
Synced w/ @davidzhanghp and we decided that keeping this is fine.